.github/workflows/stale.yml

```
@@ -0,0 +1,29 @@
+name: Close stale issues and PRs
```

```suggestion
name: Stale
```
Nitpick but I think it might look better in the UI as `Stale / Label issues and PRs`